docs/index.rst

```diff
@@ -7,6 +7,39 @@
 Welcome to bodhi's documentation!
 =================================
 
+What is Bodhi?
+--------------
+Bodhi is a modular web-based system that facilitates the process of publishing package updates for Fedora. Bodhi is the application that manages updates for Fedora releases. It maintains a single stage of repositories by adding/updating/removing packages. It can be used to create or modify updates and overrides.
+
+Advantages of bodhi
+-------------------
+
+1.  Provides an intuitive interface for developers and release engineers to manage pushing out package updates for multiple version releases.
+2.  Helps in delivering quality packages and repository sustainment with automated testing.
+3.  Helps Community testers to get involved in testing of the package updates and provide quality feedback which would reach the packagers/developers.
+4.  Provides a modular framework that will allow future integrations to various other QA and developer tools.
+5.  Announces the arrival of New packages entering the collection.
+6.  The reasons for an update can be communicated to end users.
+7.  Generates extended update metadata which is parsed client-side by yum.
+8.  Allows security updates to be seen by users as a separate channel, with links to the security advisories.
+9.  Accomodates automated testing for updates before the latter are released.
+10. Provides pre-release test updates for end users and QA folks who wish to test these updates.
+
+Create a Bodhi development environment
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+For getting a Bodhi development environment you can refer `here <https://pypi.python.org/pypi/bodhi-client/2.3.3>`_
```

Comment:
We already have this documented in the README and in this Sphinx project. I don't think the docs home page should repeat this information.